examples/kitchensink/webpack.config.js

```diff
@@ -0,0 +1,30 @@
+var config = require('../config');
```

Comment:
for now it's cool, but at some point we need to stop depending on this externals config coming from parent directory too, if we want a scaffolder in future.